Representable/representable
# Retroactive drive add

enables a community to be added retroactively to a drive. 

A user may add a drive in the following conditions: user is logged in, viewing a community that doesn't have a drive, viewing a community they authored.

The drives that a user can add to are public, active drives in the same state as the community.

To test: 
1. create a community without a drive
2. once submitted retroactively add to a drive
3. confirm from drive dashboard that the community is added 
4. confirm that the community can not be added to anymore drives
